contributions/opening-and-assessing-issues.md

```diff
@@ -109,6 +109,12 @@ GitHub [milestones](https://docs.github.com/en/issues/using-labels-and-milestone
 
 Before a new milestone can be created in the repository, the new milestone must be approved by the project Maintainers, following the decision making rules in the [GOVERNANCE.md](../GOVERNANCE.md) file.
 
-### GitHub project assignments
+## GitHub project assignments
 
 In place of assigning a milestone, consider assigning issues to a GitHub [project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects). An issue can have multiple project assignments, and each Project Maintainer can manage their own GitHub projects and the priorities within those projects.
+
+## Critical priority issues
+
+Such a priority must be used sparingly and only applies to released package issues that severely impact a large number of users. For example, an issue that results in data loss for all users can be considered critical, while an issue causing compile errors for some developers does not warrant a critical priority.
```

Comment:
Can we replace these with examples relevant to this project? I'm not sure anybody's ever had data loss from MRTK...

Reply:
I struggle with an example here.   However, I don't foresee a critical issue in our future.   The support team here at Microsoft wanted to make sure we have a plan in case a security issue should arise.   

Happy for you to add a better example here.

Reply:
> That's sort of the point :) Critical type things should be rare/unlikely.

I hope I'm misunderstanding this statement, but (the way I'm reading it) I'm not sure I agree. Aspirationally, sure, we should make sure we don't push out critical problems often. But we shouldn't _define_ critical events around things that are unlikely to happen. Criticality should be based on severity, not frequency.

The easiest example, based around the existing example (assuming at this point that it's not changing regardless of feedback), could be something like:

> For example, an issue that results in **_compile errors_** for all users can be considered critical, while an issue causing compile errors for some developers does not warrant a critical priority.

Hopefully you agree that an unbuildable MRTK is a critical issue 😉

Reply:
Oh, I see that the examples were removed in the latest iteration. Fair enough!